docs/ref/contrib/gis/gdal.txt

```diff
@@ -27,6 +27,8 @@ __ http://www.gdal.org/ogr/
 Overview
 ========
 
+.. _sample_data:
```

Comment:
try to use a less generic title to prevent conflicts with other docs, e.g. gdal_sample_data